#! /bin/sh /usr/share/dpatch/dpatch-run
## gnome-free-gdu.dpatch by Kilian Krause <kilian@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: backport from trunk to make gnome-autogen.sh happy

@DPATCH@
diff -urNad ekiga-2.0.11~/configure.in ekiga-2.0.11/configure.in
--- ekiga-2.0.11~/configure.in	2007-09-18 09:38:25.000000000 +0000
+++ ekiga-2.0.11/configure.in	2007-10-06 18:11:37.000000000 +0000
@@ -219,6 +219,30 @@
 libname="lib"  
 ;;
 esac
+dnl ###############################
+dnl   Gnome-Doc-Utils
+dnl ###############################
+GDU="disabled"
+if test "x${gm_platform}" != "xmingw" ; then
+  AC_ARG_ENABLE(gdu, [  --disable-gdu			Disable GNOME-Doc-Utils support],,enable_gdu=yes)
+
+  if test "x$enable_gdu" = "xyes"; then
+GNOME_DOC_INIT
+    AC_PATH_PROG(SK_CONFIG,scrollkeeper-config,no)
+    if test "x$SK_CONFIG" = "xno"; then
+      AC_MSG_ERROR(Couldn't find scrollkeeper-config. Please install the scrollkeeper package: http://scrollkeeper.sourceforge.net)
+    fi
+    GDU="enabled"
+  else
+    ENABLE_SK_TRUE="#"
+    ENABLE_SK_FALSE=""
+  fi
+else
+  ENABLE_SK_TRUE="#"
+  ENABLE_SK_FALSE=""
+fi
+
+AM_CONDITIONAL(HAVE_GNOME_DOC_UTILS, test "x${enable_gdu}" = "xyes")
 
 
 dnl ###########################################################################
